src/server/routing/router-public.ts

```diff
@@ -164,6 +166,34 @@ router.get(
   }
 );
 
+// /**
+//  * Zaak status endpoint redirects to zaakstatus if authenticated, else redirect to login
+//  */
+router.get(BffEndpoints.ZAAK_STATUS, zaakStatusHandler);
+
+export async function zaakStatusHandler(
+  req: Request,
+  res: Response,
+  next: NextFunction
+) {
+  try {
+    const authProfileAndToken = getAuth(req);
+    const params = new URLSearchParams(req.url);
+    if (authProfileAndToken) {
+      const redirectUrl = `${AppRoutes.ZAAK_STATUS}${params ? `?${params}` : ''}`;
+      return res.redirect(process.env.MA_FRONTEND_URL + redirectUrl);
+    } else {
+      if (params.get('auth-type') === 'eherkenning') {
+        return res.redirect(authRoutes.AUTH_LOGIN_EHERKENNING);
+      } else {
+        return res.redirect(authRoutes.AUTH_LOGIN_DIGID);
+      }
```

Comment:
Kom je ook `AppRoutes.ZAAK_STATUS` terecht na inloggen?

Reply:
aaah zo

Reply:
als het goed is werkt het nu